docs/cli/defradb_client.md

```diff
@@ -1,11 +1,11 @@
 ## defradb client
 
-Interact with a running DefraDB node as a client
+Interact with a DefraDB node
```

Comment:
Todo: I still see this being `DefraDB node` and other instance saying `database node`

Reply:
✅